src/metabase/types.cljc

```diff
@@ -66,6 +66,20 @@
 (derive :entity/SubscriptionTable :entity/GenericTable)
 (derive :entity/EventTable :entity/GenericTable)
 
+;;; Modifier Types
+
+;; `:type/field-values-unsupported` enables driver developers to opt out of field values calculation for specific
+;; fields. For more details see the `driver-changelog.yml`, section `Metabase 0.50.0`.
+(derive :type/field-values-unsupported :type/*)
```

Comment:
This was already here, I just moved it. It was introduced in https://github.com/metabase/metabase/pull/40293/files#diff-f2959272d8c40a4a2f1c9d6fafce63de8089b8ac88cdf8e9d3dc90ad7fdfcc80